src/env.rs

```diff
@@ -0,0 +1,131 @@
+use std::collections::HashMap;
+use std::env;
+use std::fmt;
+
+use heck::ShoutySnekCase;
+
+/// RoverEnv allows us to mock environment variables while
+/// running tests. That way we can run our tests in parallel,
+/// and our local development environment will not have unintended
+/// side effects on our tests.
+#[derive(Debug, Clone)]
+pub struct RoverEnv {
+    mock_store: Option<HashMap<String, String>>,
+}
+
+impl Default for RoverEnv {
+    fn default() -> RoverEnv {
+        RoverEnv::new()
+    }
+}
+
+impl RoverEnv {
+    /// creates a new environment variable store
+    pub fn new() -> RoverEnv {
+        let mock_store = if cfg!(test) {
+            Some(HashMap::new())
+        } else {
+            None
+        };
+
+        RoverEnv { mock_store }
+    }
+
+    /// returns the value of the environment variable if it exists
+    pub fn get(&self, key: RoverEnvKey) -> std::io::Result<Option<String>> {
+        let key = key.to_string();
+        match &self.mock_store {
+            Some(mock_store) => {
+                println!("FROM MOCK");
```

Comment:
do you want to keep this print statement?

Reply:
hehe nope